packages/ubuntu_bootstrap/test/storage/test_storage.dart

```diff
@@ -49,5 +50,70 @@ StorageModel buildStorageModel({
   when(model.hasZfs).thenReturn(hasZfs ?? true);
   when(model.hasTpm).thenReturn(hasTpm ?? false);
   when(model.hasDd).thenReturn(hasDd ?? false);
+
+  when(model.getAllTargets()).thenAnswer((_) {
+    switch (scenario) {
+      case SecureBootScenarios.supported:
+        return [Supported];
+      case SecureBootScenarios.noTpm:
+        return [NoTpm];
+      case SecureBootScenarios.bios:
+        return [Bios];
+      case SecureBootScenarios.thirdPartyDrivers:
+        return [ThirdPartyDrivers];
+      default:
+        return [];
+    }
+  });
   return model;
```

Comment:
Could be written a bit shorter with pattern-style matching:

```suggestion
  when(model.getAllTargets()).thenAnswer((_) => switch (scenario) {
        SecureBootScenarios.supported => [Supported],
        SecureBootScenarios.noTpm => [NoTpm],
        SecureBootScenarios.bios => [Bios],
        SecureBootScenarios.thirdPartyDrivers => [ThirdPartyDrivers],
        _ => [],
      });
```

Reply:
sorry, messed up the suggestion diff in codium..

Reply:
Oh nice that's much tidier!